apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter.tsx

```diff
@@ -0,0 +1,76 @@
+import { IBaseViewModel } from '@coscrad/api-interfaces';
+import './generic-index-table-presenter.css';
+
+// TODO add me to the types library
```

Comment:
Tag this with a story. We'll wait, but we should do this fairly soon. It will clean up our entire monorepo.

Reply:
done